docs/02-get-started/04-trigger-workload-with-event.md

```diff
@@ -0,0 +1,225 @@
+---
+title: Trigger a workload with an event
+---
+
+We already know how to create and expose a workload ([Function](02-deploy-expose-function.md) and [microservice](03-deploy-expose-microservice.md)). 
+Now it's time to actually use them.
+We're going to trigger a workload with an event. 
+
+## Create a Function 
+
+For this purpose, we are going to create a new Function and expose it with an APIRule. As this is a sample Function, we'll make it so that it both sends and receives the events. 
+
+<div tabs name="Deploy a Function" group="trigger-workload">
+  <details open>
+  <summary label="kubectl">
+  kubectl
+  </summary>
+
+```bash
+cat <<EOF | kubectl apply -f -
+  apiVersion: serverless.kyma-project.io/v1alpha1
+  kind: Function
+  metadata:
+    labels:
+      serverless.kyma-project.io/build-resources-preset: local-dev
+      serverless.kyma-project.io/function-resources-preset: S
+      serverless.kyma-project.io/replicas-preset: S
+    name: lastorder
+    namespace: default
+  spec:
+    deps: '{ "dependencies": {}}'
+    maxReplicas: 1
+    minReplicas: 1
+    source: |
+      let lastEvent = {}
+      module.exports = {
+        main: async function (event, context) {
+          if (event["ce-type"]) {
+            console.log("Received event:", event.data)
+            lastEvent = event.data
+          }
+          else {
+            console.log("Returning last event: ", lastEvent)
+          }
+          
+          return lastEvent 
+        } 
+      }
+---
+  apiVersion: gateway.kyma-project.io/v1alpha1
+  kind: APIRule
+  metadata:
+    name: lastorder
+    namespace: default
+  spec:
+    gateway: kyma-gateway.kyma-system.svc.cluster.local
+    rules:
+      - path: /.*
+        methods: ["GET", "POST"]
+        accessStrategies:
+          - handler: allow
+            config: {}
+    service:
+      host: lastorder.$CLUSTER_DOMAIN
+      name: lastorder
+      port: 80
+EOF
+```
+
+If the resources were created successfully, the command returns this message:
+
+```bash
+function.serverless.kyma-project.io/lastorder created
+apirule.gateway.kyma-project.io/lastorder created
+```
+
+To check that the Function is properly exposed, call it: 
+
+```bash
+curl https://lastorder.$CLUSTER_DOMAIN
+```
+
+As we haven't sent our event yet, we expect to get an empty object `{}` in response.
```

Comment:
No :(
443 error again

![image](https://user-images.githubusercontent.com/76950046/133647995-a179e2da-be76-4560-8d82-ab6905ab286c.png)

Reply:
The local setup isn't working right now.